jenkins_python/scripts/PullRequestReport.py

```diff
@@ -134,12 +135,52 @@ def buildTestReport(templateEnv):
     print("Unit Test summary %s" % unitTestSummary)
     return failed, unitTestSummaryHTML, unitTestSummary
 
+def buildPy27Report(templateEnv):
+    py27Summary = {}
+    failed = False
+
+    try:  
+        with open('added.message', 'r') as messageFile:
+            lines = messageFile.readlines()
+        
+            if len(lines):
+                py27Summary['added.message']=lines
+                failed = True
+    except:
+        print("Was not able to open file added.message")
+
+
```

Comment:
If we ever do install a PEP-8 checker, it will complain that you should have at most one blank line here.